src/_gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -87,30 +89,35 @@ eink_anr_frei:
     5:
       upper_threshold: inf
       rate_linear: 0
-eink_anr_frei_kinder:
+parameter_anrechnungsfreies_einkommen_mit_kindern_in_bg:
   name:
-    de: Abweichende anrechnungsfreie Einkommensanteile falls Kinder im Haushalt
-    en: null
+    de: >-
+      Parameter der Berechnung des anrechnungsfreien Einkommens wenn Kinder in der
+      Bedarfsgemeinschaft leben
+    en: >-
+      Parameter for calculation of income not subject to transfer withdrawal when
+      children are in the Bedarfsgemeinschaft
   description:
     de: >-
-      Obere Einkommensgrenze bei anzurechnendem Einkommen, Intervall 3, Haushalt mit
-      Kindern. Gilt statt der Parameter in eink_anr_frei für Leistungsbezieher, die mit
-      minderjährigem Kind in BG leben oder die ein minderjähriges Kind haben (zweiteres
-      wird aktuell noch nicht im Code beachtet). § 30 SGB II. Seit 01.04.2011 § 11b SGB
-      II (neugefasst durch B. v. 13.05.2011 BGBl. I S. 850. Artikel 2 G. v. 24.03.2011
-      BGBl. I S. 453).
-      Ersetzt Parameter in eink_anr_frei, falls nicht vorhanden
+      Obere Einkommensgrenze bei anzurechnendem Einkommen, Intervall 3,
+      Bedarfsgemeinschaft mit Kindern. Gilt statt der Parameter in
+      parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg für Leistungsbezieher, die
+      mit minderjährigem Kind in BG leben oder die ein minderjähriges Kind haben
+      (zweiteres wird aktuell noch nicht im Code beachtet). § 30 SGB II. Seit 01.04.2011
+      § 11b SGB II (neugefasst durch B. v. 13.05.2011 BGBl. I S. 850. Artikel 2 G. v.
+      24.03.2011 BGBl. I S. 453). Ersetzt Parameter in
+      parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg, falls nicht vorhanden
     en: null
   type: piecewise_linear
```

Comment:
This must become `require_converter` because we will get rid of the `deviation_from` method. Ignore for now.